### PR TITLE
Gitlab API token fix

### PR DESCRIPTION
**Pull request by dwoods**
This pull request fixes an issue where settings the gitlabAPIToken helm value causes an error rendering the chart yaml.  The {{ include }} for the secret name fails due to it being in a 'with' block.

Closes: #262

### CODE REVIEW

File: charts/lagoon-core/Chart.yaml [hunk 1]
```diff
@@ -5,9 +5,9 @@ description: >
 home: https://github.com/uselagoon/lagoon-charts
 icon: https://raw.githubusercontent.com/uselagoon/lagoon-charts/main/icon.png
 maintainers:
-- name: smlx
-  email: scott.leggett@amazee.io
-  url: https://amazee.io
+  - name: smlx
```
Comment (smlx) [inline charts/lagoon-core/Chart.yaml:8]: I think this might be a mistake.. could we change it back?

**Review by smlx — CHANGES_REQUESTED** [review]

File: charts/lagoon-core/Chart.yaml (hunk 1 above)
Comment (smlx) [reply charts/lagoon-core/Chart.yaml:8]: otherwise looks good!

Comment (dwoods) [reply charts/lagoon-core/Chart.yaml]: Reverted changes, thanks!

**Comment by smlx** [comment]
hey @dwoods happy to merge this if you can bump the chart version again

**Comment by dwoods** [comment]
Bumped chart version

**Comment by smlx** [comment]
Thanks @dwoods!  :tada:

**Pull request merged**